ci/repokitteh/modules/newcontributor.star

```diff
@@ -14,7 +14,9 @@ def get_pr_author_association(issue_number):
     path="repos/envoyproxy/envoy/pulls/%s" % issue_number)["json"]["author_association"]
 
 def is_newcontributor(issue_number):
-  return get_pr_author_association(issue_number) == "FIRST_TIME_CONTRIBUTOR"
+  return (
+    get_pr_author_association(issue_number)
+    in ["NONE", "FIRST_TIME_CONTRIBUTOR", "FIRST_TIMER"])
```

Comment:
Looking at https://developer.github.com/v4/enum/commentauthorassociation/:
I think there are 2 other possible associations to consider: COLLABORATOR and MEMBER

If the author happens to be a first time contributor and also a member of the Envoy org, would get_pr_author_association return FIRST_TIME_CONTRIBUTOR or MEMBER?  Similarly past the first contribution, would it return CONTRIBUTOR or MEMBER?

Reply:
Will wait to hear from @phlax on this one though I think we can also just see how it goes with this. Up to you both.

/wait-any

Reply:
> If the author happens to be a first time contributor and also a member of the Envoy

afaict - and it doesnt seem 100% clear, but yep my ~understanding is that if you never committed to envoy before you could have 3 possible designations for author assoc:

- NONE - no association
- FIRST_TIMER - no previous commits on github, but member
- FIRST_TIME_CONTRIBUTOR - member with prev commits  on github

after you contribute for the first time you go to  either `MEMBER` or `CONTRIBUTOR`

this might be wrong, but i havent been able to find any better docs

the one i have tested is `NONE` as i cant add/remove members to the org to test the others

Reply:
>  I think we can also just see how it goes with this. 

yep. Its hard to test this specifically because once you rk-bless the pr is already raised

Reply:
Merging the change and seeing how it goes sounds good to me.